components/org.wso2.micro.integrator.extensions/org.wso2.micro.integrator.management.apis/src/main/java/org/wso2/micro/integrator/management/apis/security/handler/JWTTokenSecurityHandler.java

```diff
@@ -21,19 +21,28 @@
 import org.apache.commons.logging.Log;
 import org.apache.commons.logging.LogFactory;
 import org.apache.synapse.MessageContext;
+import org.wso2.micro.core.util.CarbonException;
 import org.wso2.micro.integrator.management.apis.Constants;
+import org.wso2.micro.integrator.management.apis.ManagementApiUndefinedException;
 import org.wso2.micro.integrator.security.MicroIntegratorSecurityUtils;
 import org.wso2.micro.integrator.security.user.api.UserStoreException;
 
-import java.util.Map;
+import javax.xml.stream.XMLStreamException;
+import java.io.IOException;
 import java.util.Objects;
 
+import static org.wso2.micro.integrator.management.apis.Constants.USERNAME_PROPERTY;
+
 public class JWTTokenSecurityHandler extends SecurityHandlerAdapter {
 
     private static final Log LOG = LogFactory.getLog(JWTTokenSecurityHandler.class);
     private String name;
-    private Map<String, char[]> userList = null;
-    private MessageContext messageContext;
+
+    public JWTTokenSecurityHandler(String context) throws CarbonException, XMLStreamException, IOException,
+            ManagementApiUndefinedException {
+        super(context);
+        populateDefaultResources();
```

Comment:
Shall we move this to super class as this is common to all IINM

Reply:
done